class.jetpack.php

```diff
@@ -18,6 +18,7 @@
 use Automattic\Jetpack\Tracking;
 use Automattic\Jetpack\Plugin\Tracking as Plugin_Tracking;
 use Automattic\Jetpack\Redirect;
+use Automattic\Jetpack\Jetpack_User_Agent_Info;
```

Comment:
I think the `Jetpack_User_Agent_Info` should become part of the `jetpack-devices` package, and could more be renamed `Automattic\Jetpack\Device\User_Agent_Info`